formula/src/test/java/com/instacart/formula/FormulaRuntimeTest.kt

```diff
@@ -316,7 +315,7 @@ class FormulaRuntimeTest(val runtime: TestableRuntime, val name: String) {
             .output { assertThat(child.state).isEqualTo(3) }
 
         inspector.assertRunCount(1)
-        inspector.assertEvaluationCount(HasChildFormula::class, 8)
+        inspector.assertEvaluationCount(HasChildFormula::class, 5)
```

Comment:
The changes in this test class display the improvements in the number of evaluations.